packages/trpc/server/routers/viewer/teams/getTeamWithMinimalData.handler.ts

```diff
@@ -0,0 +1,51 @@
+import { getMinimalTeam } from "@calcom/lib/server/queries/teams";
+import { prisma } from "@calcom/prisma";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TGetTeamWithMinimalDataInputSchema } from "./getTeamWithMinimalData.schema";
+
+type GetTeamWithMinimalDataOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetTeamWithMinimalDataInputSchema;
+};
+
+export const getTeamWithMinimalData = async ({ ctx, input }: GetTeamWithMinimalDataOptions) => {
+  const teamMembership = await prisma.membership.findFirst({
+    where: {
+      userId: ctx.user.id,
+      teamId: input.teamId,
+    },
+  });
+
+  if (!teamMembership) {
+    throw new TRPCError({
+      code: "NOT_FOUND",
+      message: "Team membership not found",
```

Comment:
nit: Could be more user friendly
```suggestion
      message: "You are not a member of this team.",
```